python/cuml/test/test_arima.py

```diff
@@ -211,64 +239,89 @@ def get_dataset(data, dtype):
         y = pd.read_csv(
             os.path.join(data_path, "{}.csv".format(data.dataset)),
             usecols=range(1, data.batch_size + 1), dtype=dtype)
-        y_cudf = cudf.from_pandas(y)
-        lazy_data[key] = (y, y_cudf)
+        y_train, y_test = train_test_split(y, test_size=data.n_test,
+                                           shuffle=False)
+        y_train_cudf = cudf.from_pandas(y_train).fillna(np.nan)
+        y_test_cudf = cudf.from_pandas(y_test)
+        lazy_data[key] = (y_train, y_train_cudf, y_test, y_test_cudf)
     return lazy_data[key]
 
 
 def get_ref_fit(data, order, seasonal_order, intercept, dtype):
     """Compute a reference fit of a dataset with the given parameters and dtype
     or return a previously computed fit
     """
-    y, _ = get_dataset(data, dtype)
+    y_train, *_ = get_dataset(data, dtype)
     key = order + seasonal_order + \
         (intercept, data.dataset, np.dtype(dtype).name)
     if key not in lazy_ref_fit:
-        ref_model = [sm.tsa.SARIMAX(y[col], order=order,
+        ref_model = [sm.tsa.SARIMAX(y_train[col], order=order,
                                     seasonal_order=seasonal_order,
                                     trend='c' if intercept else 'n')
-                     for col in y.columns]
+                     for col in y_train.columns]
         with warnings.catch_warnings():
             warnings.filterwarnings("ignore")
             lazy_ref_fit[key] = [model.fit(disp=0) for model in ref_model]
     return lazy_ref_fit[key]
 
 
+###############################################################################
+#                              Utility functions                              #
+###############################################################################
+
+def mase(y_train, y_test, y_fc, s):
+    # TODO: fix for missing observations
```

Comment:
What’s the fix needed?

Reply:
The fix is there but I forgot to remove that comment 😳

Reply:
that's what I though, so I was confused 😆